src/components/ManagerPage/AppsList.js

```diff
@@ -34,6 +34,7 @@ import Update from 'icons/Update'
 import Trash from 'icons/Trash'
 import CheckCircle from 'icons/CheckCircle'
 
+import { FreezeDeviceChangeEvents } from './HookDeviceChange'
```

Comment:
we may want it somewhere else to be used in other parts of the app

Reply:
but the behavior only happen in Manager 🤔

Reply:
yeah nvm